boxsdk/object/base_api_json_object.py

```diff
@@ -145,6 +145,30 @@ def untranslated_fields(cls):
         """
         return cls._untranslated_fields
 
+    @classmethod
+    def _untranslate(cls, value):
+        """
+        Untranslates a given object into a dictionary.
+
+        :param value:
+            The object to untranslate.
+        :rtype value:
+            `dict`
+        :return:
+            A dictionary containing the untranslated object.
+        """
+        if isinstance(value, BaseAPIJSONObject):
+            new_value = cls._untranslate(value._response_object)  # pylint:disable=protected-access
+            return new_value
```

Comment:
It might be nice to one-line all of these instead of declaring a new variable and then instantly returning it